### ISSUE TITLE
Cut/Copy/paste menu opens in weird place

### ISSUE THREAD

**Issue by thazy** [goki/gi#163]
With mouse pointer at the blinking cursor in text document, 2-finger touchscreen tap open cut/copy/paste menu in weird place.

![img_0181](https://user-images.githubusercontent.com/30027010/47248914-20634a80-d3cb-11e8-9b31-33e64e7c8bb3.JPG)

**Comment by rcoreilly** [comment]
yep it uses the center of the widget, in this case the text view.  mouse position is not still avail at the point where the menu is popped up but could get it from window or something. lowpri.
